File: Application/CodeNB/app/src/main/java/com/example/codenb/model/LoginResultDto.kt
package com.example.codenb.model

import kotlinx.serialization.Serializable

@Serializable
data class LoginResultDto(
    val id: Int,
    val lastName: String,
    val firstName: String,
    val role: String,
    val token: String,
    val expiresIn: Int
)

File: Application/Backend/CodeNB.Model/Dtos/LoginResultDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CodeNB.Model
{
    public class LoginResultDto
    {
        public int ID { get; set; }
        public string? LastName { get; set; }
        public string? FirstName { get; set; }
        public string? Role { get; set; }
        public string? Token { get; set; }
        public int? ExpiresIn { get; set; }
    }
}
